bcda/models/models.go

```diff
@@ -482,9 +482,19 @@ func (cclfBeneficiary *CCLFBeneficiary) GetBlueButtonID(bb client.APIClient) (bl
 		return cclfBeneficiary.BlueButtonID, nil
 	}
 
+	patientIdMode := "HICN"
+	identifier :=  cclfBeneficiary.HICN
+	systemMode := "hicn-hash"
+	if utils.FromEnv("PATIENT_IDENTIFIER_MODE","HICN_MODE") == "MBI_MODE"  {
+		patientIdMode = "MBI"
+		identifier = cclfBeneficiary.MBI
+		systemMode = "hicn-mbi"
```

Comment:
This should be `mbi-hash`.  See https://github.com/CMSgov/beneficiary-fhir-data/pull/172/files

Reply:
good eye again!

Reply:
oh it looks like u actually said it here, oops my bad